test-apps/display-test-app/src/frontend/DisplayTestApp.ts

```diff
@@ -61,7 +61,7 @@ async function openIModel(filename: string, writable: boolean): Promise<IModelCo
 // @return Promise that resolves to true after signIn is complete
 async function signIn(): Promise<boolean> {
   if (!NativeApp.isValid) // for browser, frontend handles redirect. For native apps, backend handles it
-    await BrowserAuthorizationCallbackHandler.handleSigninCallback(DisplayTestApp.getAuthConfig().redirectUri);
+    await BrowserAuthorizationCallbackHandler.handleSigninCallback("http://localhost:3000/signin-callback");

```

Comment:
how should we obtain the redirect uri from the backend in production applications since we wouldn't want to hardcode it?

Reply:
this is only for the web app case (and DTA is a weird hybrid sometimes-web sometimes-mobile sometimes-electron amalgamation). For Electron, you don't need to do this - it's done in the backend.